plugins/org.jkiss.dbeaver.ext.iotdb.ui/src/org/jkiss/dbeaver/ext/iotdb/ui/internal/IoTDBUIMessages.properties

```diff
@@ -0,0 +1,43 @@
+controls_privilege_table_column_privilege_grant=Grant Option
+controls_privilege_table_column_privilege_grant_tip=Privilege grant option
+controls_privilege_table_column_privilege_name=Privilege
+controls_privilege_table_column_privilege_name_tip=Privilege name
+controls_privilege_table_column_privilege_status=Enabled
+controls_privilege_table_column_privilege_status_tip=Privilege activated
+controls_privilege_table_push_button_check_all=Check All
+controls_privilege_table_push_button_clear_all=Clear All
+
+edit_command_change_user_name=Update user
+edit_command_grant_privilege_action_grant_privilege=Grant privilege
+edit_command_grant_privilege_action_revoke_privilege=Revoke privilege
+edit_command_grant_privilege_name_grant_privilege=Grant privilege
+edit_command_grant_privilege_name_revoke_privilege=Revoke privilege
+
+editors_user_editor_abstract_load_grants=Load grants
+editors_user_editor_general_control_dba_privileges=DBA Privileges
+editors_user_editor_general_group_login=Login
+editors_user_editor_general_label_user_name=User Name
+editors_user_editor_general_service_load_catalog_privileges=Load catalog privileges
+editors_user_editor_privileges_service_load_privileges=Load privileges
+editors_user_editor_privileges_service_load_tables=Load tables
+
+dialog_connection_advanced_tab=Advanced
+dialog_connection_advanced_tab_tooltip=Advanced/custom driver properties
+dialog_connection_browse_button=Open ...
+dialog_connection_browse_button_tip=Open existing database file or directory
+dialog_connection_create_button=Create ...
+dialog_connection_create_button_tip=Create new database file or directory
+dialog_connection_database_schema_label=Database/Schema:
+dialog_connection_db_file_chooser_text=Choose database file
+dialog_connection_db_folder_chooser_message=Choose folder with database files
+dialog_connection_db_folder_chooser_text=Choose database folder
+dialog_connection_general_tab=General
+dialog_connection_general_tab_tooltip=General connection properties
+dialog_connection_host_label=Host:
+dialog_connection_jdbc_url_=JDBC URL
+dialog_connection_password_label=Password:
+dialog_connection_path_label=Path:
+dialog_connection_port_label=Port:
+dialog_connection_server_label=Sql Dialect:
+dialog_connection_test_connection_button=Test Connection ...
```

Comment:
This message is not used. Please remove it and check other messages from this file